src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,34 @@
+import { Person } from '../../types';
+import { getPreparedPerson } from '../../utils/getPreparedPeople';
+import { TablePerson } from '../TablePerson';
+
+type Props = {
+  people: Person[];
+};
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const preparedPeople = getPreparedPerson(people);
```

Comment:
It's better to prepare people in the parent component so `PeopleTable` will be a representation component and doesn't have any logic, just render UI
https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0
https://www.freecodecamp.org/news/separation-of-concerns-react-container-and-presentational-components/